src/Vts/MonteCarlo/Sources/PointAndLineSources/PointSources/CustomPointSource.cs

```diff
@@ -92,7 +94,9 @@ public ISource CreateSource(Random rng = null)
     /// location, direction and initial tissue region index.
     /// </summary>
     public class CustomPointSource : PointSourceBase
-    { 
+    {
+        private readonly DoubleRange _polarAngleEmissionRange;
+        private readonly DoubleRange _azimuthalAngleEmissionRange;

```

Comment:
@hayakawa16 Did you see these warnings based on the GitHub actions - these are new to me but I'm guessing Resharper will flag them in code also.

Reply:
Thank you for your review @lmalenfant!  This is a very good catch indeed.  Those variables are in the Base class so interesting.  Let me take a look.

Reply:
I looked at CustomLineSource and LineSourceBase as reference.  LineSourceBase does not have those variables defined.  This makes sense because CustomLineSource needs these variables locally for the GetFinalDirection.  A potential fix is to remove those fields from the PointSourceBase class. I'll see if this ripples.